db/migrate/20230509133628_add_columns_to_employee.rb

```diff
@@ -0,0 +1,14 @@
+class AddColumnsToEmployee < ActiveRecord::Migration[7.0]
+  def change
+    create_table :employees do |t|
+      t.string :last_name
+      t.string :first_name
+      t.string :middle_name
+      t.string :passport_data
+      t.date :date_of_birth
+      t.string :place_of_birth
+      t.string :home_address
+      t.date :employed_on
```

Comment:
Let's remove `employed_on ` and add `t.timestamps`. Always use `t.timestamps` in migrations for table creation.